### PR TITLE
Added general function for password-view-buttons

### LINKED ISSUE #2623: View-Password-Buttons
We have to add "view-password"-buttons at every place, where a password can be entered. 

E.g.: SMTP-password in the main configuration

### PR DESCRIPTION
**Pull request by modelrailroader**
Should be all password inputs
Closes #2623 

Happy Christmas!

### CODE REVIEW

File: phpmyfaq/assets/src/utils/password.js [hunk 1]
```diff
@@ -14,16 +14,18 @@
...
-  const password = document.querySelector('#faqpassword');
-
-  if (togglePassword) {
-    togglePassword.addEventListener('click', () => {
-      const type = password.getAttribute('type') === 'password' ? 'text' : 'password';
-      password.setAttribute('type', type);
-      togglePassword.classList.toggle('is-active');
-    });
-  }
+  var passwordInputs = document.querySelectorAll('input[type="password"]');
```
Comment (thorsten) [inline phpmyfaq/assets/src/utils/password.js:17]: We should use let/const instead of var.

File: phpmyfaq/admin/instances.php [hunk 2]
```diff
@@ -209,9 +209,14 @@ class="btn btn-danger pmf-instance-delete"
               <label class="col-form-label col-lg-4" for="password">
                   <?= Translation::get('ad_instance_password') ?>:
               </label>
-              <div class="col-lg-8">
-                <input class="form-control mb-2" type="password" autocomplete="off" name="password" id="password" required>
-              </div>
+                <div class="col-lg-8">
+                    <div class="input-group mb-2">
+                        <input class="form-control" type="password" autocomplete="off" name="password" id="password" toggle="instances_password_toggle" required>
```
Comment (thorsten) [inline phpmyfaq/admin/instances.php:214]: What's this toggle attribute? :-)

File: phpmyfaq/admin/instances.php [hunk 3]
```diff
@@ -209,9 +209,14 @@ class="btn btn-danger pmf-instance-delete"
...
+                        <span class="input-group-text" id="instances_password_toggle">
```
Comment (thorsten) [inline phpmyfaq/admin/instances.php:215]: Please use dashes instead of underscore.

File: phpmyfaq/admin/instances.php (hunk 2 above)
Comment (thorsten) [reply phpmyfaq/admin/instances.php:214]: We should use e.g. "data-pmf-toggle-password"

**Review by thorsten — CHANGES_REQUESTED** [review]

**Comment by modelrailroader** [comment]
I solved all requested changes.

**Review by thorsten — APPROVED** [review]

**Pull request merged**